FrontEnd/src/components/Taskoverview.js

```diff
@@ -16,9 +18,16 @@ const useStyles = makeStyles((theme) => ({
     root: {
```

Comment:
Wollen wir die Styles evtl auslagern?

Reply:
Wollten wir eh irgendwann mal machen, ja. 

Können wir dafür ein Extra Issue machen? damn kümmere ich mich darum in einem extra Thread